internal/ingress/controller/nginx.go

```diff
@@ -1090,6 +1118,21 @@ func createOpentracingCfg(cfg ngx_config.Configuration) error {
 	return os.WriteFile("/etc/nginx/opentracing.json", []byte(expanded), file.ReadWriteByUser)
 }
 
+func createOpentelemetryCfg(cfg ngx_config.Configuration) error {
+
+	tmpl, err := template.New("otel").Parse(otelTmpl)
+	if err != nil {
+		return err
+	}
+	tmplBuf := bytes.NewBuffer(make([]byte, 0))
+	err = tmpl.Execute(tmplBuf, cfg)
+	if err != nil {
+		return err
+	}
+
+	return os.WriteFile("/etc/nginx/opentelemtry.toml", tmplBuf.Bytes(), file.ReadWriteByUser)
```

Comment:
shouldn't the path come from cfg.OpentelemetryConfig?

Reply:
thanks, yeah fixed.